data/sql/customizations.sql

```diff
@@ -262,3 +262,47 @@ JOIN compound_crs c ON gt.target_crs_code = c.code AND gt.target_crs_auth_name =
 JOIN geodetic_crs gcrs ON gt.source_crs_auth_name = gcrs.auth_name AND gt.source_crs_code = gcrs.code
 JOIN vertical_crs vcrs on vcrs.auth_name = c.vertical_crs_auth_name AND vcrs.code = c.vertical_crs_code
 WHERE method_auth_name = 'EPSG' AND method_code = '9635' AND gt.deprecated = 0;
+
+ INSERT INTO "grid_transformation" VALUES(
```

Comment:
why moving that record ? It was fine in grid_transformation_custom.sql . Otherwise customizations.sql becomes a bit of a kitchen sink